docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst

```diff
@@ -0,0 +1,104 @@
+List of available Apache Kafka Connect connectors
+=================================================
+
+The following Apache Kafka Connect connectors are currently available, and can
+be used in any Aiven for Kafka services with Kafka Connect enabled. 
+
+
+Source connectors
+-----------------
+
+Source connectors enable the integration of data from an existing technology into an Apache Kafka topic. The following is the list of available connectors:
```

Comment:
"list of available connectors:" I'd change. " list of available source connectors:"